src/test/java/training/article/getArticle.feature

```diff
@@ -0,0 +1,28 @@
+Feature:  get an article
+
+  Background:
+    * url url
+    * def responseGetArticle = read('training/article/responsesuccessfully.json')
+
+  @getArticle
+  Scenario:  get an article
+
+    * def slug = passedSlug
+
+    Given path 'articles', slug
+    And header Authorization = token
+    When method get
+    Then status 200
+    And match  response == responseGetArticle
+    And assert response.article.slug == slug
+
+  @getAllArticles
+  Scenario:  get all articles
+
+    * def responseGetArticles = read('training/article/responsearticles.json')
+
+    Given path 'articles'
+    And header Authorization = token
+    When method get
+    Then status 200
+    And match  response == responseGetArticles
```

Comment:
add an space line at the end of the file, and you can check if the `articlesCount` is the same of the array (`articles`) length.